### ISSUE TITLE
Cloudflare 1020

### ISSUE THREAD

**Issue by vanpana** [oleh2208/etoro-api#10]
I'm getting 403 for login with some cloudflare error saying "Please enable cookies". Have you encountered something like this?

**Comment by oleh2208** [comment]
i'll take a look later this day

________________________________
От: Dorinel Panaite <notifications@github.com>
Отправлено: 24 августа 2020 г. 11:23
Кому: oleh2208/etoro-api <etoro-api@noreply.github.com>
Копия: Subscribed <subscribed@noreply.github.com>
Тема: [oleh2208/etoro-api] Cloudflare 1020 (#10)


I'm getting 403 for login with some cloudflare error saying "Please enable cookies". Have you encountered something like this?

**Comment by oleh2208** [comment]
any chanse you have 2factor auth?

**Comment by vanpana** [comment]
No, I have 2factor disabled. I cannot seem to reproduce it again, it happened in the first run, then it worked properly.

**Comment by oleh2208** [comment]
ok, it's probably some race condition on strart, i need to add some checks to fix it

________________________________
От: Dorinel Panaite <notifications@github.com>
Отправлено: 24 августа 2020 г. 23:22
Кому: oleh2208/etoro-api <etoro-api@noreply.github.com>
Копия: oleh2208 <oleh_22_08@outlook.com>; Comment <comment@noreply.github.com>
Тема: Re: [oleh2208/etoro-api] Cloudflare 1020 (#10)


No, I have 2factor disabled. I cannot seem to reproduce it again, it happened in the first run, then it worked properly.

**Comment by slim16165** [comment]
I have the same issue ("Please enable cookies")

**Comment by ok24601** [comment]
hi make sure you are using latest version of this project, this issue was fixed when i started using chrome instead of phantom.js
